components/artwork_columns/artwork.jade

```diff
@@ -24,3 +29,6 @@
           p.bid-now-link Bid now
         else
           p.bid-now-link Auction closed
+
+    if artwork.get('inquireable')
```

Comment:
We were actually never displaying 'Contact Gallery' link in this view at all, easy enough to add.